### PR TITLE
Done with dishesRatings endpoint

### PR DESCRIPTION
**Pull request by harethriyadh**

### CODE REVIEW

File: src/controllers/__test__/clientController.test.js [hunk 1]
```diff
@@ -202,25 +203,38 @@ describe('getFavoriteDishes', () => {
...
-    await clientController.getFavoriteDishes(req, res);
+    await clientController.dishesRatings(req, res);
 
-    expect(res.status).toHaveBeenCalledWith(500);
-    expect(res.json).toHaveBeenCalledWith({ error: 'Server error' });
+    expect(res.status).toHaveBeenCalledWith(201);
+    expect(findByIdMock).toHaveBeenCalledWith('mockDishId');
+    expect(res.json).toHaveBeenCalledWith('Dish Rated Successfully');
   });
 });
```
Comment (motaz99) [inline src/controllers/__test__/clientController.test.js:239]: Why do you have this here and inside the describe?

I think you don't need it or you do?

File: src/controllers/__test__/clientController.test.js [hunk 2]
```diff
@@ -236,14 +250,68 @@ describe('getFavoriteDishes', () => {
...
+//       json: jest.fn(),
+//     };
+
+//     await chefController.getChefById(req, res);
+
+//     expect(res.status).toHaveBeenCalledWith(200);
+//     expect(Chef.findById).toHaveBeenCalledWith(req.params.id);
+//     expect(res.json).toHaveBeenCalledWith(mockChef);
+//   });
 // });
```
Comment (motaz99) [inline src/controllers/__test__/clientController.test.js]: If you couldn't do this test, it's better to delete it rather than leave it as a comment.

**Review by motaz99 — CHANGES_REQUESTED** [review]

**Review by harethriyadh — COMMENTED** [review]
done

File: src/controllers/__test__/clientController.test.js (hunk 1 above)
Comment (harethriyadh) [reply src/controllers/__test__/clientController.test.js:239]: Did not understand you, what do you mean specifically...

**Review by motaz99 — APPROVED** [review]

**Review by TesnimNakilci — APPROVED** [review]

**Pull request merged**